Plugins/BuildServerIntegration/JenkinsIntegration/JenkinsAdapter.cs

```diff
@@ -115,21 +115,21 @@ private void AddGetBuildUrl(string projectUrl)
             }
         }
 
-        public class ResponseInfo
+        private class ResponseInfo
         {
             public string Url { get; set; }
             public long Timestamp { get; set; }
             public IEnumerable<JToken> JobDescription { get; set; }
         }
 
-        public class JenkinsCacheInfo
+        private class JenkinsCacheInfo
```

Comment:
Do we need the class at all?

Reply:
Yes, it stores the latest known build time for a project url. The adapter first asks for last build time, then requests the full data with all builds if there is anything new (that is more to save the server than the processing in the plugin).

Reply:
But you maybe meant why use a class, why not long directly?
I will change to that
Making a few other touchups, want to test this on Monday/Tuesday before pshing and initiates a merge